README.md

```diff
@@ -2,79 +2,97 @@
 
 npx nuxi analyze
 
-# Env vars
-
-CI=0 or CI=1
-
 # Tests
 
-Run unit tests and component tests:
+To run unit tests, component tests and integration tests, first start the Mockoon server:
+```bash
+npx mockoon-cli start --data ./mocks/mockoon.json
+```
+
+Then, to run unit tests:
 ```bash
 npm run test:unit
 # Or with coverage:
 npm run test:unit:coverage
 ```
 
+To run integration tests:
+```bash
+npm run test:integration
+```
+
 Run server-side rendering tests:
 ```bash
 npm run test:ssr
 ```
 
 Run full-stack tests:
+
+1. Stop the Mockoon server
```

Comment:
I hope it should be intuitive since the Mockoon server is only started explicitly. It can be started in the desktop UI, or on the command line. In the former case I hope that the 'green play button' / 'red stop button' interface is intuitive, and in the latter case I imagine people can work out they need to do ctrl+C.

If the developer fails to stop Mockoon, the e2e tests will complain and remind them to do so.